### PR TITLE
CODE RUB: Order Crud Operation For Teachercontacts

### PR DESCRIPTION
**Pull request by nedataghiparast**

### CODE REVIEW

File: OtripleS.Web.Api/Services/Foundations/UserContacts/UserContactService.cs [hunk 1]
```diff
@@ -33,33 +33,34 @@ public partial class UserContactService : IUserContactService
...
+            UserContact mayBeUserContact =
                 await this.storageBroker.SelectUserContactByIdAsync(userId, contactId);
 
-            ValidateStorageUserContact(storageUserContact, userId, contactId);
+            ValidateStorageUserContact(mayBeUserContact, userId, contactId);
 
-            return storageUserContact;
+            return await this.storageBroker.DeleteUserContactAsync(mayBeUserContact);
         });
+
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Services/Foundations/UserContacts/UserContactService.cs:64]: Remove this line

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by nedataghiparast** [comment]
@ShriHumrudha Done Please take another look

File: OtripleS.Web.Api/Services/Foundations/UserContacts/UserContactService.cs (hunk 1 above)
Comment (ShriHumrudha) [reply OtripleS.Web.Api/Services/Foundations/UserContacts/UserContactService.cs:64]: I still see the line 64

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by ShriHumrudha** [comment]
Closing due to inactivity.

**Pull request closed without merging**